docs/source/build.rst

```diff
@@ -76,8 +76,8 @@ Prerequisites
 
 - CMake 3.26.4+
 - GCC 9.3+ (11.4+ recommended)
-- CUDA Toolkit 11.4+
-- Volta architecture or better (compute capability >= 7.0)
+- CUDA Toolkit 12.4+
```

Comment:
Should we use CUDA 12.0+ given that is what we document in RAPIDS and test in CI?

```suggestion
- CUDA Toolkit 12.0+
```

Reply:
Good catch. For some reason I thought 12.4 was the lower bound.